lms/static/sass/course/courseware/_courseware_search.scss

```diff
@@ -85,6 +85,13 @@
       bottom: lh(.75);
       text-transform: uppercase;
     }
+    .search-results-ellipsis {
+      &:after {
+        content: '\2026';
```

Comment:
Added ellipsis as a content after the span element provided on the edx-search side.